rpc/block_svc_routines.c

```diff
@@ -41,7 +41,8 @@
                                 "'%s' ls | grep -e tpg -e '%s' | grep -B1 '%s' | grep -o 'tpg\\w'"
 # define   GB_CHECK_PORTAL      "targetcli /iscsi/" GB_TGCLI_IQN_PREFIX \
                                 "'%s' ls | grep '%s' > " DEVNULLPATH
-# define   GB_SAVECONFIG_SO_CHECK "grep -m 1 '\"config\":.*%s\",' " GB_SAVECONFIG " > " DEVNULLPATH
+# define   GB_SAVECFG_GBID_CHECK "grep -m 1 '\"config\": .*%s\",' " GB_SAVECONFIG " > " DEVNULLPATH
```

Comment:
@lxbsz 
I like this.

To be more precise should we emphasis   ``` '\"config\": .*/block-store/%s\",' ``` ?

Reply:
Yeah, sure. Will update it right now.
Thanks.

Reply:
@lxbsz this is still not reflecting in the PR. Thanks!